src/ray/protobuf/gcs.proto

```diff
@@ -140,6 +143,46 @@ message ActorTableData {
   TaskSpec task_spec = 14;
 }
 
+message PlacementGroupTableData {
+    // State of a placement group.
+    enum PlacementGroupState {
+      // Placement Group is pending or scheduling
+      PENDING = 0;
```

Comment:
Does it means it goes back to PENDING if reconstructing?

Reply:
Yep, if a placement group creating failed, it will be added into the tail of the queue and the status will keep PENDING